src/main/plugins/org.dita.base/build_preprocess_template.xml

```diff
@@ -54,7 +54,15 @@ See the accompanying LICENSE file for applicable license.
     
     <basename property="dita.input.filename" file="${args.input}" />
```

Comment:
Nevermind...I'll need to submit another pull request for the changes. I'll talk to Martin at the office for a more generic solution. Thanks!